.bazelrc

```diff
@@ -53,7 +49,9 @@ build:ios_lldb_test --define=apple.experimental.tree_artifact_outputs=1
 # - iOS multi arch test configuration
 build:ios_multi_arch_test --config=ios
 build:ios_multi_arch_test --ios_minimum_os=10.2
-build:ios_multi_arch_test --ios_multi_cpus=i386,x86_64
+# i386 was removed on rules_apple 3.x.x - this test case needs reworking
+# to exemplify fat binaries on the latest version
+build:ios_multi_arch_test --ios_multi_cpus=x86_64
```

Comment:
Would adding `arm64` to the list suffice?

Reply:
I tried that and it failed with an unknown architecture issue, i think potentially theres a bug in `transition_support.bzl` within rules_ios we need to look at after this